### PR TITLE
Add support for Nabox wallet

### PR DESCRIPTION
**Pull request by therealemjy**
## Jira ticket(s)

VEN-1389

## Changes

- add support for Nabox wallet

### CODE REVIEW

File: src/translation/translations/en.json [hunk 1]
```diff
@@ -97,28 +97,21 @@
...
-    "remainingTime": "<Icon/> Remaining time:\u00a0<White>{{date, distanceToNow}}</White>",
     "successfulConvertTransactionModal": {
       "title": "Convert submitted successfully"
     },
-    "vrtEqualsXvs": "1 VRT = {{xvsToVrtConversionRatio}} XVS",
     "withdraw": "Withdraw",
     "withdrawableAmount": "Withdrawable amount",
-    "withdrawXvs": "Withdraw XVS",
-    "xvsAVailable": "Current XVS Available",
-    "youWillReceive": "You will receive"
```
Comment (gleiser-oliveira) [inline src/translation/translations/en.json:119]: @therealemjy Just confirming you want to take these copy changes here, they don't seem related to the wallet

File: src/translation/translations/en.json [hunk 2]
```diff
@@ -166,7 +156,6 @@
       "block": "Block",
       "created": "Created",
       "from": "From",
-      "id": "ID",
```
Comment (gleiser-oliveira) [inline src/translation/translations/en.json:169]: Same

File: src/translation/translations/en.json (hunk 1 above)
Comment (therealemjy) [reply src/translation/translations/en.json:119]: Yeah these were automatically deleted when I ran the `extract-translation-keys` script. I double checked and they aren't used anymore.

File: src/translation/translations/en.json (hunk 2 above)
Comment (therealemjy) [reply src/translation/translations/en.json:169]: Same thing here, this was automatically deleted and I couldn't find any place that uses it

**Review by gleiser-oliveira — APPROVED** [review]
